### DISCUSSION TITLE
Is there a reason why links starting with # in markdown get taget=blank?

### DISCUSSION THREAD

**Discussion by ResuBaka** [histoire-dev/histoire#234] [Ideas]
When you want to have internal links in markdown docs with for example `[Color](#color)` it gets target blank added.

Is there a reason that we do not exclude links starting with `#` in: https://github.com/histoire-dev/histoire/blob/e6b60febd67d657daa00e629441e50132c0ad531/packages/histoire/src/node/markdown.ts#L39

**Comment by Akryum** [comment]
Not really :sweat_smile:

**Reply by ResuBaka** [reply]
Should I open a MR to fix it or do you want to fix it?

**Reply by Akryum** [reply]
Feel free to open a PR! :ok_hand:
